eZ/Publish/Core/Repository/ContentService.php

```diff
@@ -1929,6 +1955,18 @@ public function loadRelations(APIVersionInfo $versionInfo)
             throw new UnauthorizedException('content', $function);
         }
 
+        return $this->internalLoadRelations($versionInfo);
+    }
+
+    /**
+     * Loads all outgoing relations for the given version without checking the permissions.
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException if the user is not allowed to read this version
```

Comment:
This is no longer true, what is true though:

```suggestion
     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
```